README.md

```diff
@@ -34,14 +34,15 @@ module.exports = {
         pingbacks: false,
         forwardPingbacksAsWebmentions: "https://example.com/endpoint",
         domain: "example.com",
+        fetchLimit: 10000, // number of webmentions to fetch
```

Comment:
is there something important about this number specifically?

Reply:
it’s the default